### DISCUSSION TITLE
Send data from device via http request

### DISCUSSION THREAD

**Discussion by Jumbo125** [Waboodoo/HTTP-Shortcuts#346] [Q&A]
Hy developers
It looks like a great app!

I'm searching for a app, how can send some device infos like ram, cpu, temp, automatically to a local adress. is this possible with this app?
i need it automatically. for example:
send all 5 seconds a http request with infos: ram, cpu, temp, battery.

Can i do this with your app?
is it easy?

thanks a lot

**Comment by Waboodoo** [comment]
Hello,

The app is capable of sending HTTP requests automatically and regularly, but it can't read out the type of data that you want. This this usecase you might be able to use a general automation app such as [Tasker](https://play.google.com/store/search?q=tasker), [Macrodroid](https://play.google.com/store/search?q=macrodroid) or [Automate](https://play.google.com/store/apps/details?id=com.llamalab.automate), to name a few. I don't know whether these apps can give you all the information you need, but it might be a starting point.
